tests/storage-vm

```diff
@@ -45,6 +45,10 @@ for poolDriver in $poolDriverList; do
                 [ "$(lxc storage get "${poolName}" size)" = "20GiB" ]
         fi
 
+        # Ensure non-power-of-two sizes are rounded appropriately (most relevant for zfs)
```

Comment:
Can we please only execute this if the `${poolDriver}` is not `powerflex`? In PowerFlex valid volume sizes are always multiples of `8GiB`. It will fail the volume creation if other sizes like `13GB` are provided.

Reply:
It's just that when using PowerFlex you basically cannot create a volume with sizes different than 8, 16, 24, 32, 40, .... PowerFlex's API will return an error.

So when running `lxc storage volume create "${poolName}" roundVolSize --type=block size=13GB` PowerFlex will complain.

Reply:
Sure... but the other drivers round their inputs silently instead of letting the backing driver error out (e.g. 16KiB ZFS block boundaries)

Reply:
The difference is that at the size of gigabytes the rounding is neglible in comparison to the allocated size, whereas with powerflex if i ask for 1gib i would actually get 8gib, which is noticeable. So we made the decision to report it to the user rather than round.

Reply:
How much rounding should LXD do before failing? Isn't it with LVM that big'ish rounding is also needed to meet the "full PE" count?

Reply:
Certainly not Gigabytes